osxcollector/output_filters/opendns/api.py

```diff
@@ -56,18 +57,56 @@ def categorization(self, domains):
             A dict of {domain: categorization_result}
         """
         url_path = 'domains/categorization/?showLabels'
-        response = self._requests.multi_post(self._to_url(url_path), data=simplejson.dumps(domains))
-        response = response[0]
+        all_responses = {}
+
+        if self._cache:
+            api_name = 'opendns-categorization'
```

Comment:
That's generally "not pythonic".
In this case I put BASE_URL up top so it is really obvious but for the others it doesn't seem so necessary to make them so obvious.